### DISCUSSION TITLE
Error deploing dcm4chee-arc-ear-5.26.0-psql.ear: Unsatisfied dependencies for type XRoadServiceProvider

### DISCUSSION THREAD

**Discussion by victorqedu** [dcm4che/dcm4chee-arc-light#3669] [Q&A]

**Comment by vrindanayak** [comment]
@victorqedu Please provide some stack trace. Looking at the previous removed descriptions - have you correctly deployed and installed the dcm4che libraries? - See point 3 in [Installation - Setup Wildfly](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Installation#setup-wildfly)

**Reply by victorqedu** [reply]
Thank you, I can't find the dele button, I wanted to delete de request. So I deleted the content... please ignore the message. I was using the wrong version of wildfly.
